src/GraphQL/Resolver/Video.php

```diff
@@ -129,7 +129,7 @@ public function resolveData($value = null, $args = [], $context = [], ResolveInf
                 $asset = $value->getData();
                 $fieldHelper = $this->getGraphQlService()->getAssetFieldHelper();
                 $fieldHelper->extractData($data, $asset, $args, $context, $resolveInfo);
-                $data['data'] = $data['data'] ? base64_encode($data['data']) : null;
+                $data['data'] = isset($data['data']) && !empty($data['data']) ? base64_encode($data['data']) : null;
```

Comment:
`isset` is not necessary as `empty` already checks that the variable is set :wink: